app.js

```diff
@@ -2,6 +2,9 @@
 const express = require('express');
 const app = express();
 const mongoose = require('mongoose');
```

Comment:
Always order imports alphabetically and put them on the top of the page. `const app = express(); ` is just instantiation of `express` so it needs to go under imports separated by a blank line.
I know this is out of this PR's scope, but it's bugging me so please make it right. 🙈